runtime/v2/runc/v2/service.go

```diff
@@ -159,7 +159,7 @@ func readSpec() (*spec, error) {
 	return &s, nil
 }
 
-func (s *service) StartShim(ctx context.Context, id, containerdBinary, containerdAddress, containerdTTRPCAddress string) (string, error) {
+func (s *service) StartShim(ctx context.Context, id, containerdBinary, containerdAddress, containerdTTRPCAddress string) (address string, retErr error) {
```

Comment:
If we only care error, we can use the format like `(_ string, retErr error)`. WDYT?

Reply:
variable address appears below - we are not just returning error.

I think the current form in PR is more readable.

Reply:
yeah. but in some condition, we just return empty string. That is why I think we don't need address as named var.

Reply:
```
runtime/v2/shim/shim.go:
                address, err := service.StartShim(ctx, idFlag, containerdBinaryFlag, addressFlag, ttrpcAddress)
```
From the usage, it seems the caller expects address.
It seems naming the return value accordingly is better.

Reply:
I agree with @fuweid here. This function is complex enough and named returns doesn't make things easier. So avoiding collisions with variables inside sounds like a good idea.

Reply:
@mxpv 
Wei has given lgtm.
I think the naming of address variable is minor (in the scope of this fix).
So I would keep it if possible.

Reply:
Named return var is used for documentation basically. We can put more information in interface side. And according to the change, `address` named return var is not helpful because just two return statements are using `address`. It is not readable actually.

Reply:
@mxpv @fuweid 
I have hid the name for address return value.